manifest.yml

```
@@ -0,0 +1,24 @@
+name : "AWS_IoT_Device_SDK_for_Embedded_C"
+version: "202009.00"
+description: |- 
+  "The AWS IoT Device SDK for Embedded C is a collection of C source
+  files that can be used in embedded applications to securely connect 
+  to AWS IoT Core and interact with other AWS IoT services on AWS Cloud.\n
+  \n
+  See dependencies for included libraries."
+dependencies:
+  - name: "coreMQTT"
+    version: "v1.0.0"
+      type: "git"
```

A row for `repository` is missing
```suggestion
    version: "v1.0.0"
    repository:
      type: "git"
```